Dockerfile

```diff
@@ -84,12 +84,26 @@ RUN make -C ${BUILD_BASE}/tools/sys-utils/ -j$(nproc) all
 # ------------------------------------------------------------------------------
 FROM c-builder as libcmt-debian-packager
 ARG CMT_BASE=${BUILD_BASE}/tools/sys-utils/libcmt
-ARG TOOLS_LIBCMT=libcmt.deb
+ARG VERSION=0.0.0
 USER root
-RUN make -C ${CMT_BASE} debian-package \
-    TARGET_DESTDIR=${BUILD_BASE}/_install \
-    TARGET_PREFIX=/usr/riscv64-linux-gnu \
-    LIBCMT_DEB_FILENAME=${BUILD_BASE}/${TOOLS_LIBCMT}
+
+RUN make -C ${CMT_BASE} \
+        ARG_VERSION=${VERSION} \
+        TARGET_PREFIX=/usr \
+        TARGET_DESTDIR=${BUILD_BASE}/install/run \
+        install-run libcmt-v${VERSION}.deb
```

Comment:
Is the convention to have the "v" in the version? Same for the others.

Reply:
That is what we do in `machine-emulator`, e.g. https://github.com/cartesi/machine-emulator/releases/download/v0.18.0-test1/cartesi-machine-v0.17.0_amd64.deb